scripts/zones/King_Ranperres_Tomb/mobs/Vrtra.lua

```diff
@@ -30,6 +30,8 @@ entity.onMobSpawn = function(mob)
 end
 
 entity.onMobEngaged = function(mob, target)
+    mob:setLocalVar("twohourTime", 0)
```

Comment:
Can condense this into just mob:resetLocalVars()

Reply:
> Can condense this into just mob:resetLocalVars()

Ah true, are there some underlying local vars (from other scripts) that should not be reset though? like some resist building or whatever? I am always worried about that.

Reply:
Hmm true. Don't mobs reset everything when they regen to 100% HP? We could put a check in the roam calls and reset a mob's local vars if it reaches 100%.

Reply:
> Hmm true. Don't mobs reset everything when they regen to 100% HP? We could put a check in the roam calls and reset a mob's local vars if it reaches 100%.

Ah, I do not think the currently reset, I think local vars only reset on spawn. I would be worried that some things are coded with that assumption and thus it breaks some things. Maybe I keep this as is for now, it is also more expressive in terms of understanding the mob logic, good to know about resetLocalVars though!

Reply:
I agree with your thinking. There has to be a few mobs that rely on logic set on spawn, and require it persist until death, etc. etc. But yeah! resetLocalvars can be used in a plethora of places for sure.